modules/party/plugins/usrmgr_details_main.php

```diff
@@ -6,20 +6,25 @@
 
 $user_party = $db->qry_first('
```

Comment:
It would be good to implement pending refactoring changes (as here switching from $db to $database) whenever we touch code. A test for function has to be done anyways

Reply:
Fixed in https://github.com/lansuite/lansuite/pull/914/commits/519b4f653232c790ea0b3f3e215173055f7b4c70